tests_metricflow/query_rendering/test_time_spine_join_rendering.py

```diff
@@ -152,3 +152,34 @@ def test_join_to_time_spine_with_queried_time_constraint(
         dataflow_plan_builder=dataflow_plan_builder,
         query_spec=query_spec,
     )
+
+
+@pytest.mark.sql_engine_snapshot
+def test_join_to_time_spine_with_input_measure_constraint(
```

Comment:
Will you add the same test or a similar one to the check query tests? i.e., the ones in `itest_***.yaml`. I always like to add both because those ones ensure that the query can also execute (in case we miss something when reading the SQL snapshots).